node.gyp

```diff
@@ -715,6 +714,15 @@
                 'deps/http_parser/http_parser.gyp:http_parser'
               ]
             }],
+            [ 'node_shared_nghttp2=="false"', {
+              'dependencies': [
+                'deps/nghttp2/nghttp2.gyp:nghttp2'
+              ],
+              'defines': [
+                # We're using the nghttp2 static lib
+                'NGHTTP2_STATICLIB'
+              ]
```

Comment:
Can you remove the define?  That's taken care of now that #15487 has been merged.